sway/tree/container.c

```diff
@@ -181,12 +362,39 @@ struct sway_container *container_output_create(
 	return output;
 }
 
-struct sway_container *container_workspace_create(
-		struct sway_container *output, const char *name) {
-	if (!sway_assert(output,
-			"container_workspace_create called with null output")) {
-		return NULL;
+static struct sway_container *get_workspace_initial_output(const char *name) {
+	struct sway_container *parent;
+	// Search for workspace<->output pair
+	int i, e = config->workspace_outputs->length;
+	for (i = 0; i < e; ++i) {
```

Comment:
No need to declare `parent` and `i` before used.

Reply:
I didn't write this lol